admin/controllers/pupil-pin.js

```diff
@@ -62,12 +63,14 @@ const postGeneratePins = async (req, res, next) => {
   try {
     submittedPupils = await pinGenerationService.generatePupilPins(pupilsList)
     await pupilDataService.updateMultiple(submittedPupils)
-    school = await schoolDataService.findOne({_id: req.user.School})
+    school = await schoolDataService.sqlFindOneByDfeNumber(req.user.School)
```

Comment:
I was under the impression we are keeping the naming conventions for the methods for consistency. Aren't we developing this in parallel with the mongo layer?

Reply:
Almost, however we are not going to write a mongo -> sql translation layer, so generic calls like `findOne` will become as many individual queries as needed.

At the moment all new calls are being prefixed by `sql` so we build them up alongside the Mongo calls.

Reply:
im finding the distinction very useful when knee deep in coding.  You know what you've changed and what you havent